src/scaife-widgets/_variables.scss

```diff
@@ -0,0 +1,11 @@
+$white: #fff;
```

Comment:
I don't think we need this since an expected set of variables should exist at root that we should document.